keyboards/sofle/keymaps/killmaster/rules.mk

```diff
@@ -0,0 +1,21 @@
+
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
+COMMAND_ENABLE = no         # Commands for debug and configuration
+MIDI_ENABLE = no            # MIDI controls
+AUDIO_ENABLE = no           # Audio output on port C6
+UNICODE_ENABLE = no         # Unicode
+BLUETOOTH_ENABLE = no       # Enable Bluetooth with the Adafruit EZ-Key HID
+SWAP_HANDS_ENABLE = no      # Enable one-hand typing
+OLED_DRIVER_ENABLE = yes
+ENCODER_ENABLE = yes
+CONSOLE_ENABLE = no
+EXTRAKEY_ENABLE = yes
+VIA_ENABLE = no
+LTO_ENABLE = yes
+RGBLIGHT_ENABLE = no
+RGBLIGHT_DRIVER = WS2812
+RGB_MATRIX_ENABLE = yes
+RGB_MATRIX_DRIVER = WS2812
+SPLIT_KEYBOARD = yes
+MOUSEKEY_ENABLE = no
+POINTING_DEVICE_ENABLE = no
```

Comment:
```suggestion
EXTRAKEY_ENABLE = yes
LTO_ENABLE = yes
RGB_MATRIX_ENABLE = yes
RGB_MATRIX_DRIVER = WS2812
```

Reply:
Cleaned up the file and tested it.
Worked perfectly.
Thank you for the suggestion!